### ISSUE TITLE
[Bug] Default Switch story lacks a label

### ISSUE THREAD

**Issue by khmakoto** [microsoft/fluentui#22816]
_Found during Switch a11y review (#21986)._

It is being flagged by accessibility insights for web:

![image](https://user-images.githubusercontent.com/7798177/166588181-dd9ec517-d44d-4f1a-a151-df9d4f3462dd.png)
